app/connectors/CreateIncomeSourceConnector.scala

```diff
@@ -31,45 +31,66 @@ class CreateIncomeSourceConnector @Inject()(val http: HttpClient,
                                            )(implicit val ec: ExecutionContext) {
 
 
-  def addBusinessDetailsUrl(mtdItid: String): String = s"${appConfig.itvcProtectedService}/income-tax-view-change/create-income-source/business/$mtdItid"
+  def createBusinessIncomeSourcesUrl(mtdItid: String): String =
+    s"${appConfig.itvcProtectedService}/income-tax-view-change/create-income-source/business/$mtdItid"
 
   def createBusiness(mtdItid: String, createBusinessIncomeSourcesRequest: CreateBusinessIncomeSourceRequest)
-                    (implicit headerCarrier: HeaderCarrier): Future[Either[CreateBusinessErrorResponse, List[AddIncomeSourceResponse]]] = {
+                    (implicit headerCarrier: HeaderCarrier): Future[Either[CreateIncomeSourcesErrorResponse, List[CreateIncomeSourcesResponse]]] = {
```

Comment:
can we change createIncomeSources -> createIncomeSource. We are only creating one of them (at a time) so the plural here is misleading a bit.